pkg/tnf/interactive/oc.go

```diff
@@ -40,6 +40,8 @@ type Oc struct {
 	container string
 	// namespace of the pod
 	namespace string
+	// serviceAccountName of the pod
```

Comment:
is this the same thing David is adding?

Reply:
I did not want to keep passing the serviceAccountName between tests, if you [look here](https://github.com/test-network-function/test-network-function/pull/296/files/58bdf5dc2b0b3c55745d332b8e81b20436cc4eb8#diff-8f2feaab4634f4a165085b1c4b5248d7d792ed5ce4655b5652e354cdf33d3770R183)
serviceAccount is first retrieved, then passed as  an extra param to subsequent tests..
if @edcdavid  is going to clean this up, then I don't mind it being removed

Reply:
wow, it is indeed super ugly the way it was handled before with the pointer passed into multiple functions. @edcdavid, do you mind replacing this with the one you added when you pick up this change?